schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/ClinicalReportRD.avdl

```diff
@@ -67,15 +60,5 @@ protocol ClinicalReportsRD {
         */
         union {null, array<string>} supportingEvidence;
 
-        /**
-        This map should contains the version of the different DBs used in the process
-        */
-        map<string> referenceDatabasesVersions;
-
-        /**
-        This map should contains the version of the different software in the process
-        */
-        map<string> softwareVersions;
-
```

Comment:
I think we still need this because the databases and software versions used in the decision support system will have impact on the results and therefore constitute part of our provenance chain.